app/services/sdr/repository.rb

```diff
@@ -50,11 +50,12 @@ def self.register(cocina_object:, assign_doi: false)
 
     # @param [String] druid the druid of the object
     # @raise [Error] if there is an error initiating accession
-    def self.accession(druid:)
+    def self.accession(druid:, user_name:)
```

Comment:
Can you add user_name to the method docs?

Reply:
@peetucket AFAIK, it's globally available within a given thread. `Current.user` is set in both jobs touched in this PR.

Reply:
pushed a new commit to do this, and tested in localhost without error, but not sure how to verify (in localhost) that DSA got the expected param